### ISSUE TITLE
The patch size used for the validation set affects the experimental results.

### ISSUE THREAD

**Issue by yiyiyi36** [ge-xing/Diff-UNet#45]
On the BTCV dataset, when I set the validation patch size to 96,96,96 as well, the training becomes very unstable and I can't reach the Dice scores reported in the paper. What is the reason for this? @ge-xing
